### PR TITLE
Reduce the maximum amount of jobs to 3 to reduce indexing pressure

### PR DESCRIPTION
**Pull request by rinatkhaziev**
## Description

Reduce the maximum amount of jobs to 3 to reduce indexing pressure.

Don't log the fact the processor job couldn't be scheduled as it's correct behavior in case the concurrency limit is hit. 

### Plugin Updated: Enteprise Search

We've reduced the maximum number of concurrent jobs to prevent indexing operations potentially affecting search performance. 

## Checklist

Please make sure the items below have been covered before requesting a review:

- [ ] This change works and has been tested locally (or has an appropriate fallback).
- [ ] This change works and has been tested on a Go sandbox.
- [ ] This change has relevant unit tests (if applicable).
- [ ] This change has relevant documentation additions / updates (if applicable).
- [ ] I've created a changelog description that aligns with the provided examples.

### CODE REVIEW

File: tests/search/includes/classes/queue/test-class-cron.php [hunk 1]
```diff
@@ -252,11 +252,11 @@ public function configure_concurrency_data() {
 			],
 			[
 				20,
-				[ 'vip_search_queue_processor' => 5 ],
+				[ 'vip_search_queue_processor' => 3 ],
 			],
 			[ // max of 5 takes over
```
Comment (rebeccahum) [inline tests/search/includes/classes/queue/test-class-cron.php:257]: Minor nitpick, can we update the comment to reflect the change of 3?

File: search/includes/classes/queue/class-cron.php [hunk 2]
```diff
@@ -46,7 +46,7 @@ class Cron {
 	/**
 	 * The maximum number if processor jobs allowed at one time
 	 */
-	const MAX_PROCESSOR_JOB_COUNT = 5;
+	const MAX_PROCESSOR_JOB_COUNT = 3;
```
Comment (rebeccahum) [inline search/includes/classes/queue/class-cron.php:49]: Just wondering, but will this have much of an impact of it evaluates to 2.5 usually? https://github.com/Automattic/vip-go-mu-plugins/blob/90ae5a0b46c0231fce4384075f314a5cb20f7613/search/includes/classes/queue/class-cron.php#L86-L89

Comment (rinatkhaziev) [reply search/includes/classes/queue/class-cron.php:49]: I spot-checked some heavy hitters, they have much bigger concurrency, which kind of adds up to the problem.

File: tests/search/includes/classes/queue/test-class-cron.php (hunk 1 above)
Comment (rinatkhaziev) [reply tests/search/includes/classes/queue/test-class-cron.php]: thanks for the catch, addressed!

**Review by rebeccahum — CHANGES_REQUESTED** [review]
Thanks for the changes. Just one more thing, can you update the test here to reflect the new value:

https://github.com/Automattic/vip-go-mu-plugins/blob/90ae5a0b46c0231fce4384075f314a5cb20f7613/tests/search/includes/classes/queue/test-class-cron.php#L225-L226

**Comment by rinatkhaziev** [comment]
Sure thing, funny nothing failed though.

**Review by rebeccahum — DISMISSED** [review]
thxu

**Review by rebeccahum — APPROVED** [review]
gg

**Pull request merged**